src/app/shared/routing/item-route.ts

```diff
@@ -43,7 +43,7 @@ export function appDefaultItemRoute(cat: ItemTypeCategory = 'activity'): ItemRou
  * Url (as string) of the item route without attemptId or path (only item id)
 */
 export function incompleteItemStringUrl(id: ItemId): string {
-  return `/items/by-id/${id}/details`;
+  return `/${itemTypeCategoryPrefix('activity')}/by-id/${id}/details`;
```

Comment:
meaning it will not work for skills 🤔? when you see such things, you should open a new issue directly.

Reply:
I added the category as a new argument